homeassistant/components/mysensors/text.py

```diff
@@ -0,0 +1,60 @@
+"""Provide a text platform for MySenSors."""
+from __future__ import annotations
+
+from homeassistant.components.text import TextEntity
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import Platform
+from homeassistant.core import HomeAssistant, callback
+from homeassistant.helpers.dispatcher import async_dispatcher_connect
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from .. import mysensors
+from .const import MYSENSORS_DISCOVERY, DiscoveryInfo
+from .device import MySensorsEntity
+from .helpers import on_unload
```

Comment:
Preference: I probably would have named this async_on_unload

Reply:
Yeah, when all the platforms are migrated to config entry setup, I'll remove this and just use the builtin `entry.async_on_unload`.